CRM/Report/Form/Contribute/Summary.php

```diff
@@ -495,14 +505,17 @@ public function from($entity = NULL) {
                         {$this->_aliases['civicrm_phone']}.is_primary = 1)";
 
     $this->addAddressFromClause();
-    if (!empty($this->_params['batch_id_value'])) {
+    //for contribution batches
```

Comment:
There is a nice function isTableSelected which works well in places like this - it's a bit less fragile - ie. if someone then adds 'order by' the isTableSelected 'get's that.